### DISCUSSION TITLE
How to install NeuralNet?   With opentrack2022.3.13

### DISCUSSION THREAD

**Discussion by proxlamus** [opentrack/opentrack#1556] [Q&A] [answered]
Hey guys.  I'm having trouble finding Neuralnet in the Tracker input category.   It's not available in the drop down with the latest 2022.3.13 update.  

So I downloaded the Neuralnet folder and dropped it in the windows file folders.    I  also noticed it's missing the .dll for the module.  

 Can someone point me on the right direction on how to use this in windows 10?

**Comment by sthalik** [comment, marked as answer]
It's `onnxruntime.dll` in opentrack's directory.

To get a detailed error message that is helpful in diagnosing this, drag opentrack's executable or shortcut to an empty `cmd.exe` window and press Enter.

**Comment by proxlamus** [comment]
That was outrageously fast !     I just downloaded the 2022.3.2 and it works great!   The Neuralnet tracker is in the drop down and its perfect!

Thank you so much!       Is there a way I can donate to the project?

**Reply by sthalik** [reply]
The project doesn't receive donations. Just enjoy it.
